src/main/resources/META-INF/rust-core.xml

```diff
@@ -540,6 +540,11 @@
                          enabledByDefault="true" level="ERROR"
                          implementationClass="org.rust.ide.inspections.RsMainFunctionNotFoundInspection"/>
 
+        <localInspection language="Rust" groupName="Rust"
+                         displayName="Detached file inspection"
```

Comment:
`Inspection` word is useless here because this name is already shown in inspection settings